tests/unit/test.program.js

```diff
@@ -326,6 +326,20 @@ describe('program.main', () => {
       });
   });
 
+  it('passes the url of a firefox binary when specified', () => {
+    const fakeCommands = fake(commands, {
+      run: () => Promise.resolve(),
+    });
+    return execProgram(
+      ['run', '--start-url', 'www.example.com'],
```

Comment:
Thanks, this extra test will be helpful. However, I think it will become redundant once we do https://github.com/mozilla/web-ext/issues/377